resume.html

```diff
@@ -82,18 +82,22 @@
         <h1>Employment</h1>
 
         <div class="entry">
-          <div class="margin">Apr 11 - Current</div>
+          <div class="margin">Apr 11 - May 15</div>
           <div class="body">
             <span class="name"><a href="http://metamarkets.com/">Metamarkets</a></span>
             <span class="where">San Francisco, CA</span><br>
             Software Engineer
             <ul>
-              <li>Makin' it work.</li>
+              <li>Built out the Metamarkets product that took the company through four rounds of funding.</li>
```

Comment:
Do you also want to talk about data visualization related insights you brought to the product?